packages/components/src/mobile/segmented-control/index.native.js

```diff
@@ -140,8 +140,8 @@ const SegmentedControls = ( {
 		styles.selectedDark
 	);
 
-	const width = segmentsDimensions[ activeSegmentIndex ].width;
-	const height = segmentsDimensions[ activeSegmentIndex ].height;
+	const width = segmentsDimensions[ activeSegmentIndex ]?.width;
+	const height = segmentsDimensions[ activeSegmentIndex ]?.height;
```

Comment:
Was this optional chain guarding against a specific scenario that you encountered?

Reply:
I had issues with this being `undefined`, as this is depending on `onLayout` callbacks, I wouldn't be surprised if we have some issues in Sentry related to this 😅